### ISSUE TITLE
Load more than 40 toots in Mastodon timeline 

### ISSUE THREAD

**Issue by Ezelty** [TwidereProject/Twidere-Android#1023]
## Expected behavior
Twidere should load the number of toots you set as limit per load and allow to load more toots if necessary. For example, if the items limit per load is set to 100 and 160 toots have been posted after the last time you loaded toots, it should load 100 toots and allow to load more toots.

## Actual behavior
Twidere loads only 40 toots, whereas I set the items limit per load to 200. It doesn't allow to load more toots.
The only moment when I can load more than 40 toots is when there are no toots stored in the database.

## Steps to reproduce
Load toots when more than 40 toots have been posted.

## Extra info
I don't use auto refresh.
This behavior is only with Mastodon, with both my accounts, but not with my Twitter accounts.
To get around the issue, I recently tried and put the limit to 35: Twidere loaded 35 toots and proposed to load more, and I was able to see all the toots.

Android version: (version code here)
7.1.1

Network type:

- [x] Cellular
- [x] Wi-Fi

App version: (version code here)

Build variant:

- [ ] Google
- [x] F-Droid

Micro-blogging service:

- [ ] Twitter
- [ ] Fanfou
- [ ] GNU Social / StatusNet
- [x] Mastodon

App settings:

- [ ] Custom API
- [ ] Custom consumer key/secrets
- [ ] [Official keys](https://gist.github.com/mariotaku/5465786)
- [ ] Using [Thumbor](https://github.com/thumbor/thumbor)
- [ ] Connected through proxy
- [ ] Media preview enabled

**Comment by georgedorn** [comment]
>  It doesn't allow to load more toots.

I just noticed this behavior yesterday.  Even since turning off my Twitter feed (a month ago), I have been missing hundreds of toots without noticing.  Gaps in the Twitter timeline would prompt the "Load more tweets" button, but Mastodon-only timelines never get that button.

**Comment by ehashman** [comment]
I have encountered the same bug, confirmed that this is still an issue.

**Comment by Tedko** [comment]
Cant reproduce on myside.
@trwnh @ehashman @georgedorn @Ezelty can you reproduce this issue?

**Comment by ehashman** [comment]
@Tedko yep, same reproduce. Log in with Mastodon, load latest posts, don't get a "Load more..." button for more than 40 toots per account.
